pandas/io/html.py

```diff
@@ -236,48 +241,58 @@ def _text_getter(self, obj):
         """
         raise AbstractMethodError(self)
 
-    def _parse_td(self, obj):
-        """Return the td elements from a row element.
+    def _equals_tag(self, obj, tag):
+        """Returns whether an individual DOM node matches a tag
 
         Parameters
         ----------
         obj : node-like
+            A DOM node.
+
+        tag : string
+            Tag to be checked for equality
 
         Returns
         -------
-        columns : list of node-like
-            These are the elements of each row, i.e., the columns.
+        boolean
+            Does the object match tag 'tag'?
         """
         raise AbstractMethodError(self)
 
-    def _parse_tables(self, doc, match, attrs):
-        """Return all tables from the parsed DOM.
+    def _contains_tag(self, obj, tag):
+        """Returns whether an individual DOM node has a particular tag
+        contained within it
 
         Parameters
         ----------
-        doc : tree-like
-            The DOM from which to parse the table element.
+        obj : node-like
```

Comment:
Let's try to re-specify as something other than node-like.

Reply:
I totally copied "node-like" from other parts of the documentation. Suggestion of what I should use?

Reply:
@jowens : See discussion <a href="https://github.com/pandas-dev/pandas/pull/17089#discussion_r137319069">above</a>.  I don't want to buck convention if this has been the thing to use.  However, the best thing you can do is try to determine the data-type yourself when you call the function.  For example, add a `print(type(obj))` in your implementation to see what it is.

Reply:
Just provide the data-type that you get when you add `print(type(obj))` to the implementation of the function.  I'm not entirely sure what it is myself, hence why I suggested you do that to get an exact answer.